src/olympia/addons/serializers.py

```diff
@@ -459,6 +460,19 @@ class ESAddonSerializerWithUnlistedData(
     previews = ESPreviewSerializer(many=True, source='all_previews')
 
 
+class ESAddonAutoCompleteSerializer(ESAddonSerializer):
+    class Meta(ESAddonSerializer.Meta):
+        fields = ('id', 'icon_url', 'name', 'url', )
+        model = Addon
+
+    def get_url(self, obj):
+        # Addon.get_url_path() wants current_version to exist, but that's just
+        # a safeguard. We don't care and don't want to fetch the current
+        # version field to improve perf, so give it a fake one.
+        obj._current_version = Version()
```

Comment:
just so that I said it: I feel like that should be workaroundable in `current_version` but I don't see an easy way to do that just yet, so 👍 for this here for now. It's documented at least :)